server/index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const getImages = require('./model.js');
+const app = express();
+const port = 3003;
+
+// app.use(express.static('public'));
+
+app.get(`/api/images/:id`, function(req, res){
+  var params = [];

```

Comment:
Since you're using `const` in other places, I think it would be best if you avoided `var` and just used `let` or `const` everywhere.